src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,24 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import classNames from 'classnames';
+import { UserInfo } from '../UserInfo';
+import './TodoInfo.scss';
+
+export const TodoInfo = ({ todo }) => {
+  const {
+    completed,
+    title,
+    user,
+  } = todo;
+
+  return (
+    <article className={classNames(
```

Comment:
All is good. However, you can use a shortcut for className (cn) as Misha did in his video, besides, you can move a static class to the top to make it stand out like:
```
className=(cn('TodoInfo', {
      'TodoInfo--completed': completed,
    })}
```

Although, your option is also great, and easy to read)

Reply:
➕